ktor-template-core/build.gradle

```diff
@@ -30,36 +30,65 @@ artifacts {
 
 publishing {
     publications {
-        ArtifactPublication(MavenPublication) {
-            from components.java
+        maven(MavenPublication) {
```

Comment:
We might be able to move all this into the root `build.gradle`

Reply:
I'll try it out, but I'm not sure if it publishes a single artifact or three artifacts, let me try it out.

Reply:
As expected, it produces three artifacts, one per module:

![image](https://user-images.githubusercontent.com/803668/127007444-ed1ec2af-41fa-4fef-a325-b27b671c6d3b.png)